Core.UnitTests/CodeGeneration/ReflectionEmit/ModuleBuilderFactoryTest.cs

```diff
@@ -96,23 +110,35 @@ public void CreateModuleBuilder_StrongNamed_ProvidedKey ()
           c_assemblyName, assemblyDirectoryOrNull: null, strongNamed: true, keyFilePathOrNull: otherKeyPath);
 
       var publicKey = new StrongNameKeyPair (File.ReadAllBytes (otherKeyPath)).PublicKey;
-      CheckAdapterAndSaveToDiskBehavior (result, _currentDirectory, expectedPublicKey: publicKey);
+
+      CheckAdapterBehavior (result, expectedPublicKey: publicKey);
+#if FEATURE_ASSEMBLYBUILDER_SAVE
+      CheckSaveToDiskBehavior (result, _currentDirectory);
+#endif
     }
 
-    private void CheckAdapterAndSaveToDiskBehavior (IModuleBuilder moduleBuilder, string assemblyDirectory, byte[] expectedPublicKey = null)
+    private void CheckAdapterBehavior (IModuleBuilder moduleBuilder, byte[] expectedPublicKey = null)
     {
       Assert.That (moduleBuilder, Is.TypeOf<ModuleBuilderAdapter>());
       var moduleBuilderAdapter = (ModuleBuilderAdapter) moduleBuilder;
       Assert.That (moduleBuilderAdapter.ScopeName, Is.EqualTo (c_assemblyFileName));
       Assert.That (moduleBuilderAdapter.AssemblyBuilder, Is.TypeOf<AssemblyBuilderAdapter>());
       var assemblyBuilderAdapter = (AssemblyBuilderAdapter) moduleBuilder.AssemblyBuilder;
+
       Assert.That (assemblyBuilderAdapter.AssemblyName, Is.EqualTo (c_assemblyName));
       Assert.That (assemblyBuilderAdapter.PublicKey, Is.EqualTo (expectedPublicKey ?? new byte[0]));
+    }
+
+    private void CheckSaveToDiskBehavior (IModuleBuilder moduleBuilder, string assemblyDirectory)
+    {
+      Assert.That (moduleBuilder, Is.TypeOf<ModuleBuilderAdapter>());
+      var moduleBuilderAdapter = (ModuleBuilderAdapter) moduleBuilder;
+      Assert.That (moduleBuilderAdapter.ScopeName, Is.EqualTo (c_assemblyFileName));
+      Assert.That (moduleBuilderAdapter.AssemblyBuilder, Is.TypeOf<AssemblyBuilderAdapter>());
+      var assemblyBuilderAdapter = (AssemblyBuilderAdapter) moduleBuilder.AssemblyBuilder;
 
       var assemblyPath = Path.Combine (assemblyDirectory, c_assemblyFileName);
       var pdbPath = Path.Combine (assemblyDirectory, c_pdbFileName);
-      Assert.That (File.Exists (assemblyPath), Is.False);
-      Assert.That (File.Exists (pdbPath), Is.False);
```

Comment:
Please re-add those two lines. Removing them would be an unrelated change and you'd have to consider if the event of a left-over file could result in a false positve or strange exception.

Reply:
I'm afraid I don't quite understand. In the last review you suggested moving the declaration of `assemblyPath` and `pdbPath` to `CheckSaveToDiskBehavior`. In the course of doing so, these two assertions also moved into this method (lines 140/141 in the most current commit). Or am I misunderstanding something?

Reply:
I have to admit that I had to actively search for these two lines in the GitHub diff view by expanding some collapsed lines - but I knew they were there, otherwise I would not have done so probably.

Reply:
Oh no. I simply can't read and distinguish `true` from `false`. sorry

Reply:
don't know how those poor lines got lost :( but now they are back and the spam is over.